config/config.go

```diff
@@ -58,30 +62,45 @@ func (c *Config) Ensure(fromCloud bool) error {
 
 	for idx := 0; idx < len(c.Remotes); idx++ {
 		if err := c.Remotes[idx].Validate(fmt.Sprintf("%s.%d", "remotes", idx)); err != nil {
-			return err
+			if c.DisablePartialStart {
+				return err
+			}
+			golog.Global().Error(err)
```

Comment:
we should make this a debug message and also prepend a message that indicates that we found an error but is opting to continue with startup/reconfigure

Reply:
Do you think changing the error log level will lead to confusion from a user. They might want all the components built but have no idea that one is not build due since the error msg is only seen in debug logs.

Reply:
I think it'll be fine because the error still shows up during reconfiguration